src/features/applicationView/utils.ts

```diff
@@ -39,6 +44,21 @@ export const mapBerthApplicationLanguageToLanguage = (
   }
 };
 
+export const mapApplicationLanguageToLanguage = (
+  applicationLanguage: BerthApplicationLanguage | WinterStorageApplicationLanguage
+): Language | null => {
+  switch (applicationLanguage) {
+    case 'FI':
+      return Language.FINNISH;
+    case 'SV':
+      return Language.SWEDISH;
+    case 'EN':
+      return Language.ENGLISH;
+    default:
+      return null;
```

Comment:
Why don't we fall back to Finnish for example?

Reply:
The function this helper approximates returned `null` and there's a testcase for that behaviour.

As I only made this change to fix a type issue I didn't want to introduce any behavioural changes.

In that sense, although the lines are new, this code isn't new. I only added support for the WinterStorageLanguage enum into it.